tests/worker/test_model_runner.py

```diff
@@ -381,3 +383,99 @@ def test_hybrid_batches(batch_size, enforce_eager, distributed_init):
     for attr_expected, attr_actual in zip(vars(attn_metadata.decode_metadata),
                                           vars(decode_meta_actual)):
         assert attr_expected[1] == attr_actual[1]
+
+
+@pytest.mark.parametrize("enforce_eager", [True, False])
+def test_hybrid_batches_with_prefix_caching(enforce_eager, monkeypatch):
+    override_backend_env_variable(monkeypatch, STR_FLASH_ATTN_VAL)
+
+    model_runner = _create_model_runner(
+        "facebook/opt-125m",
+        seed=0,
+        dtype="float16",
+        enforce_eager=enforce_eager,
+        max_num_batched_tokens=100000,
+        max_num_seqs=100000,
+        enable_chunked_prefill=True,
+        enable_prefix_caching=True,
+    )
+
+    seq_lens: List[int] = []
+    seq_group_metadata_list: List[SequenceGroupMetadata] = []
+    block_tables = {0: list(range(128))}
```

Comment:
Can you note the block size here for convenience?

Reply:
Fixed。